src/Viewer.js

```diff
@@ -335,44 +335,67 @@ Viewer.prototype.domElement = function() {
  *        constructor.
  * @return {Scene}
  */
-Viewer.prototype.createScene = function(opts) {
+Viewer.prototype.createScene = function(MLopts) {
   // TODO: set textureStore size to 0 for video scenes somehow?
 
-  opts = opts || {};
+  MLopts = MLopts || {};
 
   var stage = this._stage;
+  var layers = [];
+
+  // Multilayer options now passed as Array of opts, wrapping in Array in case for single layer argument for backward compatibility
+  if( Object.prototype.toString.call( MLopts ) !== '[object Array]' ) MLopts = Array(MLopts);
+
+  MLopts.forEach(function(opts) {
+    
+    var source = opts.source;
+    var geometry = opts.geometry;
+    var view = opts.view;
+    var textureStore = new TextureStore(geometry, source, stage, opts.textureStore);
+    
+    var layer = new Layer(stage, source, geometry, view, textureStore, opts.layerOpts);
+  
+    layers.push(layer);
+  
+    if (opts.pinFirstLevel) {
+      layer.pinFirstLevel();
+    }
+    // Added option to pin specified level, keeping pinFirstLevel for backwards compatibility
```

Comment:
Remove the `opts.pinLevel` option for now. (It might be a good idea, but let's leave it for a later time.)

Reply:
I am using the pinLevel in my multilayer implementation, because in my second layer, I am getting always preview.png loaded, which is bad quality - it's pinned as first level, even tough set as fallbackOnly in my array. I guess I am not forwarding this variable in my code here:

```javascript
var source = new Marzipano.ImageUrlSource(function(tile) {
	if(tile.z === 0) { // preview = no tiles
		var y = defaultCubeMapFaceOrder.indexOf(tile.face) / 6;
		return {
			url: "tiles/"+mapLayer.sourceDir+"/preview.png",
			rect: { x: 0, y: y, width: 1, height: 1/6 }
		};
	} else if (tileIsTransparent(tile, levelsData)) { // level > 1
		return {
			url: "tiles/empty.png",
			rect: { x: 0, y: 0, width: 1, height: 1 }
		};
	} else {
		return {
			url: "tiles/"+mapLayer.sourceDir+"/"+tile.z+"/"+tile.face+"_y"+tile.y+"_x"+tile.x+".png",
			rect: { x: 0, y: 0, width: 1, height: 1 }
		};
	}
});
```
Any idea where to put the fallbackOnly check?
Also, I assume that pinFirstLevel pins always first level which is not fallbackOnly?

Reply:
The `fallbackOnly` attribute is put in the level description that you pass to the Geometry constructor - see e.g. `demos/cube-multi-res/index.js`.

`pinFirstLevel` pins the first level regardless of `fallbackOnly`.

Reply:
sorry I totally overlooked it, yes I pass it to the Geometry, but still I am getting the preview pinned. 
Okay, will put `opts.pinLevel` only into my build